### DISCUSSION TITLE
[MTR05]交作業出現的問題

### DISCUSSION THREAD

**Discussion by asd10042898** [Lidemy/forum#52] [程式導師實驗計畫]
**繳交作業出現的問題**
大家好，昨天弄了很久的交作業但就是弄不好，也上網查了些資料，不知道是否有甚麼沒注意到的部分，向大家提問！

1. 照著影片的步驟做完
2. 昨天晚上在 git status 時會顯示 "On branch week1 nothing to commit, working tree clean"，但是已經更改過hw1的內容並存檔。
3.  後來有成功更改並 commit 完 push 到 GitHub，也有成功 compare & pull resquest。
4. 但在看作業時卻顯示甚麼都沒有。
5. 後來想說在更新一次內容上傳，但又回到了步驟 2 的狀況。

**圖片1**
![1223](https://user-images.githubusercontent.com/81561777/114969169-6e793c00-9eaa-11eb-98a8-623f96f892e4.PNG)
**圖片2**
![1232](https://user-images.githubusercontent.com/81561777/114969205-83ee6600-9eaa-11eb-9839-1abb04f61fba.PNG)
**圖片3**
![1233](https://user-images.githubusercontent.com/81561777/114969227-8c46a100-9eaa-11eb-9190-edc298cc8152.PNG)

謝謝大家的解惑！

**Comment by aszx87410** [comment]
1. 以後要附一下你的 repo 連結：https://github.com/Lidemy/mentor-program-5th-asd10042898
2. 從 files changed 看來，你做的事情就只是在資料夾的根目錄新增了一個叫做 hw1 的檔案

我猜測如果你真的有去改 homeworks/week1 裡面的檔案的話，你八成是改錯檔案了  
因為我看你帳號底下還有另外一個 https://github.com/asd10042898/mentor-program-5th 我猜你可能是改到這個資料夾裡面的檔案
所以才會怎麼弄都沒反應

git 不會騙人，有改有 commit 的話就一定會有紀錄

**Reply by asd10042898** [reply]
好的，謝謝老師！
我等等在試一次看看！

**Reply by asd10042898** [reply]
問題已解決！
謝謝老師！
